avm/res/search/search-service/shared-private-link-resource/main.bicep

```diff
@@ -0,0 +1,53 @@
+metadata name = 'Search Services Private Link Resources'
+metadata description = 'This module deploys a Search Service Private Link Resource.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Conditional. The name of the parent searchServices. Required if the template is used in a standalone deployment.')
+param searchServiceName string
+
+@description('Required. The name of the shared private link resource managed by the Azure Cognitive Search service within the specified resource group.')
+param name string
+
+@description('Required. The resource ID of the resource the shared private link resource is for.')
+param privateLinkResourceId string
+
+@description('Required. The group ID from the provider of resource the shared private link resource is for.')
+param groupId string
+
+@description('Required. The request message for requesting approval of the shared private link resource.')
+param requestMessage string
+
+@description('Optional. Can be used to specify the Azure Resource Manager location of the resource to which a shared private link is to be created. This is only required for those resources whose DNS configuration are regional (such as Azure Kubernetes Service).')
+param resourceRegion string = ''
+
+// =============== //
+//   Deployments   //
+// =============== //
+
+resource searchService 'Microsoft.Search/searchServices@2022-09-01' existing = {
+  name: searchServiceName
+}
+
+resource sharedPrivateLinkResource 'Microsoft.Search/searchServices/sharedPrivateLinkResources@2022-09-01' = {
+  parent: searchService
+  name: name
+  properties: {
+    privateLinkResourceId: privateLinkResourceId
+    groupId: !empty(groupId) ? groupId : null
+    requestMessage: !empty(requestMessage) ? requestMessage : null
```

Comment:
This paramerer & groupId may be incorrect. The parameter is declared as required. Please make sure that is correct and update either the parameter or this reference accordingly

Reply:
done: https://github.com/Azure/bicep-registry-modules/pull/634/commits/21a2a3a328463ccab3f64a3551740caf26ca46e4